components/UserProfile/index.tsx

```diff
@@ -32,6 +32,46 @@ export default function Profile({
         setVisible(!visible);
     };
 
+    allTags.sort((a: Tag, b: Tag) => {
```

Comment:
Misschien een idee om deze sort in een losse functie te zetten, omdat je m 2x gebruit